src/scenes/join-scene.js

```diff
@@ -77,4 +78,14 @@ export class JoinScene extends Phaser.Scene {
         console.log("Players update!");
         this.playersLabel.updateWithPlayers(playersList);
     }
+
+    // FIXME should ultimately remove
+    removeInactivePlayers() {
```

Comment:
Why does the `join-scene` need this change as well?

Reply:
It doesn't, that was just to show where the ghost players creep in. I removed it!